tracer/src/Datadog.Trace/Sampling/CustomSamplingRule.cs

```diff
@@ -15,9 +19,11 @@ namespace Datadog.Trace.Sampling
     internal class CustomSamplingRule : ISamplingRule
     {
         private static readonly IDatadogLogger Log = DatadogLogging.GetLoggerFor<CustomSamplingRule>();
-        private static readonly TimeSpan RegexTimeout = TimeSpan.FromSeconds(1);
 
         private readonly float _samplingRate;
+        private readonly bool _alwaysMatch;
+
+        // TODO consider moving toward these https://github.com/dotnet/runtime/blob/main/src/libraries/Common/src/System/Text/SimpleRegex.cs
```

Comment:
Nice find, definitely worth 👀

Reply:
Credit to @bouwkast. I just copied the comment here from another file.